tests/integration_tests/charts/api_tests.py

```diff
@@ -560,6 +560,31 @@ def test_create_chart_validate_datasource(self):
             response, {"message": {"datasource_id": ["Datasource does not exist"]}}
         )
 
+    @pytest.mark.usefixtures("load_world_bank_dashboard_with_slices")
+    def test_create_chart_validate_user_is_dashboard_owner(self):
+        """
+        Chart API: Test create validate user is dashboard owner
+        """
+        dash = db.session.query(Dashboard).filter_by(slug="world_health").first()
+        # Must be published so that alpha user has read access to dash
+        dash.published = True
```

Comment:
Won't this change break test idempotency? Do we need to rollback the change?

Reply:
No, since the fixture being used to create the dashboard cleans it up after each test